app-backend/tool/src/test/scala/ast/MapAlgebraASTSpec.scala

```diff
@@ -11,7 +12,7 @@ import cats.syntax.either._
 import java.util.UUID
 
 
-class MapAlgebraASTSpec extends FunSpec with Matchers {
+class MapAlgebraASTSpec extends FunSpec with Matchers with MapAlgebraCodec {
```

Comment:
This ended up being a red herring, right? This can probably be removed if I'm remembering correctly.